tutorials/hello/hello_client/src/app/modules/hello/commands/create_hello_command.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/member-ordering */
```

Comment:
you can remove it from here if you are using it already at line26

Reply:
```suggestion

```